pages/common/ssh-keyscan.md

```diff
@@ -13,3 +13,7 @@
 - Retrieve certain types of public ssh keys of a remote host:
 
 `ssh-keyscan -t {{rsa,dsa,ecdsa,ed25519}} {{host}}`
+
+- Manually update the ssh known_hosts file with the fingerprint of a given host:
```

Comment:
Should be `SSH` instead of `ssh` in this example, and in the whole document as well.

Reply:
Perhaps, but I just matched the existing page structure :P

Reply:
Though actually, here we are almost referring to `ssh` the command-line command, but SSH the protocol. In this case, I'd actually argue that it does not make sense to use capital letters.

Reply:
Right,  I missed that.

Reply:
Updated